modules/installation-configure-proxy.adoc

```diff
@@ -94,6 +103,22 @@ Only the Proxy object named `cluster` is supported, and no additional
 proxies can be created.
 ====
 
+ifdef::aws-upi,aws-restricted-upi,gcp-upi[]
```

Comment:
If you expect that the same statement will always be the same for all UPI cloud clusters, please use the same context key for each cloud. It'll make it slightly easier to add and maintain other UPI clouds, and I know that several are in the queue.